repository.py

```diff
@@ -704,7 +704,7 @@ def access(self, path):
             # couldn't parse the server name out:
             return False
 
-    def openAddress(self, address):
+    def openAddress(self, address, mode="rb"):
```

Comment:
It looks like from description and usage that the mode should be always binary.
Here you are adding an ignored parameter.